mirar/io.py

```diff
@@ -86,14 +86,14 @@ def save_mef_to_path(data_list, header_list, primary_header, path):
     hdulist.writeto(path, overwrite=True)
 
 
-def open_fits(path: str | Path) -> tuple[np.ndarray, fits.Header]:
+def open_fits(path: str | Path, **fits_open_kwargs) -> tuple[np.ndarray, fits.Header]:
```

Comment:
What are the `fits_open_kwargs` that need to be passed?

Reply:
Just remove this new kwarg and put it directly in open_fits for everyone I think